app/scripts/modules/core/pipeline/config/templates/ConfigurePipelineTemplateModal.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import {Modal, Button} from 'react-bootstrap';
+import {IPipelineTemplate} from './pipelineTemplate.service';
+const template = require('./template');
```

Comment:
Is this supposed to be a default template if none is provided to the modal? If so, maybe rename to defaultTemplate.
Or if this is just leftover cruft from testing (I see the same template being passed in to the Modal), remove it.